crates/host/src/wasmbus/mod.rs

```diff
@@ -938,121 +895,50 @@ impl Host {
             }
         });
 
-        let data_watch: JoinHandle<anyhow::Result<_>> = spawn({
-            let data = data.clone();
+        spawn({
```

Comment:
Do we need to capture this task? Looks like it's handling publishing all the events?

Reply:
It does, but it's set up to terminate itself when the host shuts down. I can store the join handle and ensure that it's stopped before exiting just to ensure that it's in fact stopped.